content/en/docs/concepts/content-placement/placement.md

```
@@ -624,31 +624,152 @@ The placement controller will give a score to each filtered `ManagedCluster` and
 
-For example, the environment has a `Placement` named placement1 in default namespace, which selects 2 `ManagedClusters`, the output would be like:
+**Note**: The namespace in the Placement JSON must match a namespace where you have configured the placement permissions.
+
+#### Example output
+
+For example, the environment has a `Placement` named placement1 in default namespace, which selects 2 `ManagedClusters`:
 
 ```bash
-$ curl -k  https://127.0.0.1:8443/debug/placements/default/placement1
-{"filteredPiplieResults":[{"name":"Predicate","filteredClusters":["cluster1","cluster2"]},{"name":"Predicate,TaintToleration","filteredClusters":["cluster1","cluster2"]}],"prioritizeResults":[{"name":"Balance","weight":1,"scores":{"cluster1":100,"cluster2":100}},{"name":"Steady","weight":1,"scores":{"cluster1":100,"cluster2":100}}]}
+$ curl -k https://127.0.0.1:9443/debug/placements/default/placement1
```

markdownlint-cli2 (0.22.0)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)